<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[OpenBrowser]]>
    </line>
    <line>
      <![CDATA[login]]>
    </line>
    <line>
      <![CDATA[CreateProject]]>
    </line>
    <line>
      <![CDATA[logout]]>
    </line>
  </reporter-output>
  <suite started-at="2023-09-15T11:32:07 IST" name="Default suite" finished-at="2023-09-15T11:34:03 IST" duration-ms="115693">
    <groups>
    </groups>
    <test started-at="2023-09-15T11:32:07 IST" name="Default test" finished-at="2023-09-15T11:34:03 IST" duration-ms="115693">
      <class name="com.actitime.testscript.CustomerModule">
        <test-method is-config="true" signature="OpenBrowser()[pri:0, instance:com.actitime.testscript.CustomerModule@6f53b8a]" started-at="2023-09-15T11:32:07 IST" name="OpenBrowser" finished-at="2023-09-15T11:33:06 IST" duration-ms="58790" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[OpenBrowser]]>
            </line>
          </reporter-output>
        </test-method> <!-- OpenBrowser -->
        <test-method is-config="true" signature="login()[pri:0, instance:com.actitime.testscript.CustomerModule@6f53b8a]" started-at="2023-09-15T11:33:06 IST" name="login" finished-at="2023-09-15T11:33:34 IST" duration-ms="27799" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[login]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="testProjectCustomer()[pri:0, instance:com.actitime.testscript.CustomerModule@6f53b8a]" started-at="2023-09-15T11:33:34 IST" name="testProjectCustomer" finished-at="2023-09-15T11:33:54 IST" duration-ms="19588" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.apache.poi.ss.usermodel.Row.getCell(int)" because the return value of "org.apache.poi.ss.usermodel.Sheet.getRow(int)" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.apache.poi.ss.usermodel.Row.getCell(int)" because the return value of "org.apache.poi.ss.usermodel.Sheet.getRow(int)" is null
at com.actitime.generic.FileLib.getexceldata(FileLib.java:24)
at com.actitime.testscript.CustomerModule.testProjectCustomer(CustomerModule.java:44)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
            <line>
              <![CDATA[CreateProject]]>
            </line>
          </reporter-output>
        </test-method> <!-- testProjectCustomer -->
        <test-method is-config="true" signature="logout()[pri:0, instance:com.actitime.testscript.CustomerModule@6f53b8a]" started-at="2023-09-15T11:33:54 IST" name="logout" finished-at="2023-09-15T11:34:03 IST" duration-ms="9219" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[logout]]>
            </line>
          </reporter-output>
        </test-method> <!-- logout -->
      </class> <!-- com.actitime.testscript.CustomerModule -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
